build_projects/dotnet-cli-build/CompileTargets.cs

```diff
@@ -69,45 +69,20 @@ public static BuildTargetResult UpdateBuild(BuildTargetContext c)
 
         // Moving PrepareTargets.RestorePackages after PackagePkgProjects because managed code depends on the
         // Microsoft.NETCore.App package that is created during PackagePkgProjects.
-        [Target(nameof(PrepareTargets.Init), nameof(CompileCoreHost), nameof(PackagePkgProjects), nameof(RestoreLockedCoreHost), nameof(PrepareTargets.RestorePackages), nameof(CompileStage1), nameof(CompileStage2))]
+        [Target(nameof(PrepareTargets.Init), nameof(RestoreLockedCoreHost), nameof(PrepareTargets.RestorePackages), nameof(CompileStage1), nameof(CompileStage2))]
```

Comment:
This is a policy for the CLI to consume the host -- i.e., you are restoring the host binaries from some source for CLI's internal use (to build SharedFX and SDK). I don't expect this to change until SDK completely gets rid of the corehost.exe and friends and as @brthor points out, SharedFX moves.